src/client/components/chart-detail-page-layout/chart-detail-page.css

```diff
@@ -0,0 +1,20 @@
+ .chart-details{
+    display: flex;
+    flex-direction: column;
+      justify-content:space-around; 
+      height:85vh; 
```

Comment:
As we discussed we should maybe look into using flexbox to determine the height.